web/sites/default/config/taxonomy.vocabulary.citywide_contract.yml

```diff
@@ -2,7 +2,7 @@ uuid: 9f07b650-d55b-47f9-bc08-cfa8ac46e87f
 langcode: en
 status: true
 dependencies: {  }
-name: 'Citywide Contract'
+name: 'Contract Category'
 vid: citywide_contract
```

Comment:
Do you want to change this machine name before we are stuck with a mismatched name?

Reply:
No, not really, because that would involve re-creating all of the taxonomy terms. Or is there a way to change the machine name without having to delete the existing taxonomy?

Reply:
You could try editing it in all the config files and reimporting. It's not usually what I would recommend as I am leery of changing config by hand, but I'm also extremely disappointed every time I see a mismatched machine name on portlandor. I think it makes the structure harder to understand for new developers—and frankly leads to me having to explain the history of a name to the team every so often—which is kinda a drag 😄

Reply:
That results in a config import error "Unexpected error during import with operation update for taxonomy.vocabulary.citywide_contract: The machine name of the Taxonomy vocabulary bundle cannot be changed." 

I hear you on the mismatched names, it's not something I like either and try to avoid it as much as possible. Unfortunately, Drupal isn't very flexible when it comes to changing those and fixing it at this point means redoing a 4 point story that wasn't part of this story's original scope.

Reply:
As much as I hate to slow you down, I would be willing to sacrifice burning up the story to get the name to be something that is as generic as the content. Please burn up the existing story. When I get the time, I'd like to look into machine naming in a bit more depth and see if there is a way to force machine names in config a bit more elegantly. 🤔

Reply:
I found a solution. There is a module called Structure Sync that allows me to export the taxonomy terms into config and import them into a new taxonomy with a correct machine name. The one catch is that this PR also removes the existing citywide_contracts taxonomy config since it's no longer needed, but it can't be removed if it has any terms. So we need to manually delete the citywide_contracts taxonomy before this changed is deployed to each environment.